Applets/Info.plist

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
-	<key>UIUserInterfaceStyle</key>
-	<string>Light</string>
```

Comment:
I suppose we shouldn't remove this

Reply:
This was from another PR from Slava. We should keep it until we design a dark mode, otherwise the app will look broken on dark mode